HPXMLtoOpenStudio/resources/hpxml.rb

```diff
@@ -3892,6 +3892,7 @@ def to_oga(doc)
       XMLHelper.add_extension(cooling_system, 'SharedLoopMotorEfficiency', @shared_loop_motor_efficiency, :float) unless @shared_loop_motor_efficiency.nil?
       XMLHelper.add_extension(cooling_system, 'FanCoilWatts', @fan_coil_watts, :float) unless @fan_coil_watts.nil?
       XMLHelper.add_extension(cooling_system, 'CoolingSeedId', @clg_seed_id, :string) unless @clg_seed_id.nil?
+      XMLHelper.add_extension(cooling_system, 'CrankcasePowerWatts', @crankcase_watts, :float, @crankcase_watts_isdefaulted) unless @crankcase_watts.nil?
```

Comment:
Shift this down or up a line so that it's not between CoolingSeedId and HeatingSeedId.